requirements.txt

```diff
@@ -1,2 +1,4 @@
-https://github.com/meaningfy-ws/ted-sws/archive/main.zip
-pymongo~=4.0.2
\ No newline at end of file
+git+https://github.com/OP-TED/ted-rdf-conversion-pipeline.git@1.0.0-beta
```

Comment:
@Dragos0000 @CaptainOfHacks 
We could consider creating a new tag that integrates the latest improvements.